.jvmopts

```diff
@@ -0,0 +1 @@
+-Xmx4G
```

Comment:
Should we pick something smaller? I personally still use 1.5 gigs.

Do you happen to know what's the precedence if `JVM_OPTS` is defined? Does the sbt launcher support `JVM_OPTS`, or is that only sbt-extras?

Reply:
I just tested locally with 1.5G and it seemed to work, so I've changed it to that.

I don't know what the precedence is. I would guess the environment variable? but I don't know. Also, it seems the environment variable may be `SBT_OPTS`? (I have very little experience with this)

Reply:
I don't know either, but let's merge regardless, it's easy to mess with further, later, if needed.

Reply:
@lrytz I checked.  sbt-launcher-package scripts doesn't support `JVM_OPTS`.